dummyserver/server.py

```diff
@@ -16,6 +16,11 @@
 import warnings
 from datetime import datetime
 
+try:
+    from ssl import CERT_OPTIONAL
+except:
+    CERT_OPTIONAL = None
```

Comment:
Alright, I'd like to ask for your thoughts, given that that this is your codebase.  `dummyserver` is currently free of imports of both `ssl` and `urllib3`.  the tornado webserver is completely initialized within dummynet, so we'll need to import one or the other here.

For the most part, tests in `test` just blindly `import ssl` - although there are a few tests that tiptoe around it, a bit.

There's also `urllib3.utils.ssl_` which has a function, `resolve_cert_reqs`, which would resolve the constant `CERT_OPTIONAL` to the correct numeric value.  `urllib3.utils.ssl_` appears to be written defensively to handle cases like a missing `ssl` module, but reading through the code I'm not 100% certain that it is safe for `CERT_*` resolution if the `ssl` module doesn't exist.

So, which do you prefer: the blind import that we use in more than half the cases in the `test` directory, or the possibly safer approach of bringing in `urllib3.utils.ssl_` into dummyserver... which would be the first dependency of dummyserver on urllib3.